test/src/e2e_vm_tests/test_programs/should_pass/language/method_on_primitives/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "method_on_primitives"
+entry = "main.sw"
+
+[dependencies]
+std = { path = "../../../../../../../sway-lib-std" }
```

Comment:
We don't need the full blown `std` here. The lightweight reduced `swayl-lib-std-assert` is enough.

Reply:
Hmmm, actually I don't see the need for any dependencies at all. We are not testing here if the `len()` method is properly implemented, but just if it is properly resolved. Thus, turning the test into `library;` without any dependencies and simply compiling
```sway
let _ = __slice(&[1u8], 0, 1).len();
```
should already prove the fix is valid.

The proposal is to remove the `assert` and all the dependencies.

Reply:
Done.